larky/src/test/resources/vgs_tests/http/test_default_response.star

```diff
@@ -0,0 +1,86 @@
+"""Unit tests for request.star"""
+load("@stdlib//unittest", "unittest")
```

Comment:
Can you sort these?
And keep them consistent ? 

Use the binding: unittest="unittest"